java/src/jmri/jmrix/openlcb/OlcbActionListBundle.properties

```diff
@@ -6,9 +6,10 @@
 
 
 jmri.jmrix.openlcb.swing.monitor.MonitorAction=Open OpenLCB Monitor
-jmri.jmrix.openlcb.swing.send.OpenLcbCanSendAction=Open OpenLCB Console
 jmri.jmrix.openlcb.swing.networktree.NetworkTreeAction=Open OpenLCB Network Config
 jmri.jmrix.openlcb.swing.eventtable.EventTableAction=Open OpenLCB Event Table
 
 jmri.jmrix.openlcb.swing.hub.HubAction=Start OpenLCB Hub
+jmri.jmrix.openlcb.swing.send.OpenLcbCanSendAction=Open OpenLCB Console
+jmri.jmrix.openlcb.swing.memtool.MemoryToolAction=Open Memory Tool
```

Comment:
I'm of two minds on this.  it would be consistent with the other menu items. But these appear in either an "OpenLCB" or an "LCC" menu depending on the configuration selected.  In one case it's redundant, but in the other it's confusing. Maybe I should remove the semi-redundant OpenLCB term from the other items instead?

Reply:
I went with consistency and added the OpenLCB.  Thanks.